content/_includes/site/hire-us.njk

```diff
@@ -1,6 +1,6 @@
 {% import 'embed.macros.njk' as embed %}
 
-<aside data-layout="cta" aria-labelledby="hire-cta-title" class="hire-us-cta" data-ccs-colors="invert">
+<aside data-layout="cta" aria-labelledby="hire-cta-title" class="hire-us-cta" data-ccs-colors="light">
```

Comment:
@SondraE I noticed that we are currently using the dark gradient on both the light and dark modes. I was out for a week but wanted to check in with you to see if that was a decision you all made while I was out as I believe when we last spoke you wanted the light gradient background on light mode and the dark gradient on dark mode.

Reply:
@stacyk I didn't intend for background to be the same in light and dark mode. The background colors should change the the mode just like the gradient text changes with the mode.

Reply:
@SondraE @stacyk 

Ok, I think this is adjusted as preferred.
- The background changes based on mode. 
- The colors are the opposite of whatever mode one is in.

Is that what we want?

Light mode:
![image](https://github.com/user-attachments/assets/fe67e18b-c992-492f-b194-a93ceb7bef7d)

Dark mode:
![image](https://github.com/user-attachments/assets/00fd3884-669d-4f0c-b22e-f433b675f927)

And if we think the link underline needs to stand out more we could swap the positions of the `--brand-gradient-end` and `--brand-gradient-start` colors

![image](https://github.com/user-attachments/assets/25517f0d-316f-4b45-a4cd-81e48e951088)

![image](https://github.com/user-attachments/assets/dbbc3f5b-dbab-42df-918f-501121c2294b)